sympy/matrices/matrices.py

```diff
@@ -299,22 +299,65 @@ def _setitem(self, key, value):
             return
 
     def copy(self):
+        """
+        Returns the copy of a matrix expression.
+
+        Examples
+        ========
+
+        >>> from sympy import Matrix
+        >>> A = Matrix(2, 2, [1, 2, 3, 4])
+        >>> A.copy()
+        Matrix([
+        [1, 2],
+        [3, 4]])
+
+        """
         return self._new(self.rows, self.cols, self._mat)
 
     def trace(self):
+        """
+        Returns the trace of a matrix expression i.e. the sum of the
+        diagonal elements.
+
+        Examples
+        ========
+
+        >>> from sympy import Matrix
+        >>> A = Matrix(2, 2, [1, 2, 3, 4])
+        >>> A.trace()
+        5
+
+        """
         if not self.is_square:
             raise NonSquareMatrixError()
         return self._eval_trace()
 
     def inv(self, method=None, **kwargs):
+        """
+        Return the inverse of a matrix using Gaussian elimination by default.
+
+        Also, by setting method='LU' we can specify it to be done using LU decomposition.
+
+        However, if determinant of the matrix is zero, a ``ValueError`` is raised as the inverse for that
```

Comment:
if determinant -> if the determinant

This line and the one above appear too long for an 80 character terminal.